File: sample/python/nested_if_else.py
def foo(condition: bool):
    if not condition:
        return
    print()
    print()


def bar(condition: bool):
    if not condition:
        return
    print(
        0,
    )


def baz(condition: bool):
    if not condition:
        return
    if condition:
        print()
        return
    print()

File: sample/javascript/nested-if-else.js
function nestedIfElse(foo) {
    if (foo === 2) {
        console.log('foo');
        return;
    }
    if (foo === 3) {
        console.log('bar');
    } else {
        console.log('baz');
    }
}

console.log(nestedIfElse);
